ledger/ledger-api-health/src/main/scala/com/digitalasset/ledger/api/health/HealthStatus.scala

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.ledger.api.health
+
+sealed trait HealthStatus
```

Comment:
This interface will be used by 3rd parties. They have raised to our attention in the past that they have problems using traits from Java implementations and asked us to use abstract classes for ADTs, to make their life's easier

Reply:
Makes sense. I take it `ReportsHealth` is fine as a trait, as it’s mixed into other traits?

Reply:
Or, as discussed earlier, we'll provide separate Scala and Java APIs to provide the best integration experience.